### ISSUE TITLE
Open questions/discussion

### ISSUE THREAD

**Issue by bsmithgall** [codeforamerica/wand#3]
Have a question? Add it here!

**Comment by ErnieAtLYD** [comment]
So we built a working json example of one node:

```
[
{
    "title":"Some Title",
    "content":"blah Blah Blah",
    "type":"pickOne",
    "triggers":[]
}
]
```

But what would two nodes look like, where node one points to node two? @mapsam? @bengolder? @creade?

**Comment by bsmithgall** [comment]
I think the json in [`index.html`](https://github.com/codeforamerica/wand/blob/608a0115f18afdd9a5b67eb699f3635654ae873f/examples/index.html#L14) has a multi-node setup.

**Comment by mapsam** [comment]
Yup, we've been working with those three nodes since our first mob programming. What we haven't discussed yet is any conditional node statements - if they answered "yes" to a previous node, answering "yes" to this one will bring them in one direction. But if they answered "no" in that same previous question, answering "yes" now goes to a different direction.

**Comment by ErnieAtLYD** [comment]
Closing this, but to keep the issue honest it's an array of objects:

```
[
  {
    "id":0,
    "title": "Some Title",
    "content": "Do you want a thing or another thing?",
    "type": "pickOne",
    "triggers": [
      {
        "content":"A thing",
        "target":1
      },
      {
        "content":"Another thing",
        "target":"FART"
      }
    ]
  },
  {
    "id":1,
    "title": "Another Title",
    "content": "You are on another page.",
    "type": "pickOne",
    "triggers": []
  },
  {
    "id":"FART",
    "title": "Third Title",
    "content": "You are on yet another third page again.",
    "type": "pickOne",
    "triggers": []
  }
]
```

**Comment by ErnieAtLYD** [comment]
@mapsam So sales force has a [really terrible wizard designer](http://userdocs.servicemax.com/ServiceMaxHelp/Summer11/en_us/CNF19.htm) for one of their enterprise solutions. But one thing that piqued my interest was one of their metadata being a `Criteria To Enable Action`. What if that we had that where we were evaluating an expression? What if it was implemented on a trigger? Or a node?
